README.md

```
@@ -66,73 +93,170 @@ docker compose up -d
-| CLI | Commander.js (`npm i -g opencodehub-cli`) |
+| Storage | Local filesystem or S3-compatible (AWS, MinIO, R2) |
+| AI | OpenAI, Anthropic, Google, Groq, Ollama, OpenRouter |
+| Queue | BullMQ + Redis |
+| CLI | Commander.js + Inquirer |
 
 ---
 
-## Documentation
+## Architecture
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)